ext/zpages/test/tracez_data_aggregator_benchmark.cc

```diff
@@ -0,0 +1,249 @@
+#include "opentelemetry/context/threadlocal_context.h"
+#include "opentelemetry/ext/zpages/tracez_data_aggregator.h"
+
+#include <benchmark/benchmark.h>
+#include <chrono>
+
+#include "opentelemetry/sdk/trace/tracer.h"
+
+using namespace opentelemetry::sdk::trace;
+using namespace opentelemetry::ext::zpages;
+using opentelemetry::core::SteadyTimestamp;
+
+/////////////////////////////// BENCHMARK HELPER FUNCTIONS //////////////////////////////
+
+/*
+ * Helper function that creates and ends i spans instantly.
```

Comment:
Please add a comment that states what isUnique does.
Nit: isUnique should be is_unique (https://google.github.io/styleguide/cppguide.html#Variable_Names)

Reply:
whoops, thanks i missed that one!!